process.md

```diff
@@ -61,126 +62,334 @@ This is for two main reasons:
-* **Alternative proposals** Even if alternatives aren't obvious,
-  forcing yourself to brainstorm a couple more approaches may give you
-  new ideas or make clear that your initial proposal is the best one
+- **The use case(s) it solves** Who needs this and why?
+- **Requirements** What needs to be true about the solution?
+- **Alternative proposals** Even if alternatives aren't obvious, forcing
+  yourself to brainstorm a couple more approaches may give you new ideas or make
+  clear that your initial proposal is the best one
 
 Also feel free to reach out to us on [slack](contact.md#slack) if you want any
 help/guidance.
 
 Thanks so much!!
 
-## OWNERS and Collaborators
+## Contributor Ladder
+
+<!--Big thanks to the folks at https://github.com/cncf/project-template
+for providing the base framework for this section.-->
+
+This contributor ladder outlines the different contributor roles within the
+project, along with the responsibilities and privileges that come with them.
+Community members generally start at the first levels of the "ladder" and
+advance up it as their involvement in the project grows. Our project members are
+happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of
+things. "Responsibilities" are things that contributor is expected to do.
+"Requirements" are qualifications a person needs to meet to be in that role, and
+"Privileges" are things contributors on that level are entitled to.
+
+### Community Participant
+
+Description: A Community Participant engages with the project and its community,
+contributing their time, thoughts, etc. Community participants are usually users
+who have stopped being anonymous and started being active in project
+discussions.
+
+- Responsibilities:
+  - Must follow the [Tekton CoC](code-of-conduct.md)
+- How users can get involved with the community:
+  - Participating in community discussions (Slack, mailing list, etc)
+  - Helping other users
+  - Submitting bug reports
+  - Commenting on issues
+  - Trying out new releases
+  - Attending community events
+
+### Contributor
+
+Description: A Contributor contributes directly to the project and adds value to
+it. Contributions need not be code. People at the Contributor level may be new
+contributors, or they may only contribute occasionally.
+
+A Contributor must meet the responsibilities and has the requirements of a
+Community Participant, plus:
+
+- Responsibilities include:
+  - Follow the project [contributing guide](CONTRIBUTING.md)
+- Requirements (one or several of the below):
+  - Report and sometimes resolve issues
+  - Occasionally submit PRs
+  - Contribute to the documentation
+  - Participate in meetings
+  - Answer questions from other community members
+  - Submit feedback on issues and PRs
+  - Test releases and patches and submit reviews
+- Privileges:
+  - Invitations to contributor events
```

Comment:
Agreed that voting has its own requirements (which is actually more in line with org member)

w.r.t. events - depends on the event, but in general I agree and would expect most events we run to be community events and not contributor events.

I included this because I know that [k8s runs contributor events at kubecon](https://www.kubernetes.dev/events/kcsna2021/) that are intended to be smaller scale fun events intended for contributors, and I could see us potentially doing something similar at cdCon. :)

Reply:
@wlynch maybe we could include something here to say that contributors maybe be eligible to vote and run in elections and link to the requirements?

Reply:
Done.